### ISSUE TITLE
Compiling MoppyDesk

### ISSUE THREAD

**Issue by don3194** [samphonic/MoppyClassic#122]
I do not know how to use NetBeans very well, and I was hoping someone could tell me how to correctly compile this into an executable file of some sort. I have tried some buttons that say "Clean and Build" or "Generate Javadoc" and neither of them seem to compile them into one file. Thanks for the Help!
Sincerely,
don3194

**Comment by samphonic** [comment]
Sorry for the incredible lag: "Clean and Build" should create a MoppyDesk.jar file in the output directory for the project.  Take a look at the output log for the file path.

**Comment by don3194** [comment]
Sorry for my incredible lag as well. So I have located the "MoppyDesk.jar" created from clicking the "Clean and Build" button, but when I click to open it, nothing happens. By the way I feel like a complete idiot right now like I am missing something too obvious.

**Comment by xenoesc** [comment]
try right clicking the file and selecting "run" worked for me

sorry for my incredible lag also

**Comment by samphonic** [comment]
Yeah, you should be able to just press the "Run" button in NetBeans and it'll run.  If no windows pop up, check the output at the bottom of the screen to see if there was an error.
